app/workers/schedule/every_morning_worker.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Schedule
+  class EveryMorningWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :schedule, retry: 0
+
+    def perform
+      User.find_each do |user|
+        @tasks = (Task.where(scope: :scope_public) || Task.where(user_id: user.id)).where.not(status: :done)
```

Comment:
```suggestion
        @tasks = Task.scope_public.or(Task.where(user_id: user.id)).not_done
```

Reply:
for scopes (and negative scopes) https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html

Reply:
or condition https://guides.rubyonrails.org/active_record_querying.html#or-conditions